src/components/SearchInput.vue

```diff
@@ -45,7 +45,12 @@
         }
       },
 
-      onBlur() {
+      onBlur(event) {
+        const result = this.props.onBlur(event)
```

Comment:
Hey, where the `this.props.onBlur()` comes from? Since this component doesn't have a prop called `onBlur`.

Reply:
Hi, 
sorry i was a bit "poking around" here. I am quite new to js, so please be a bit more patient with me ;). I ran into the mentioned issue. When using IE11, opening a treeselect with a vertical scrollbar and then click anywhere on the scrollbar ... it directly closes the treeselect. 
I derived this from half-backed google results. I apologize. But it supressed the issue in IE11 and i wanted to share the information. 
I am actually fine when you reject the pull request. The problem is (hopefully) nearby the point in code.
Sorry for inconvenience ;)

Reply:
Never mind :)
I am getting very close to have this bug fixed. Thanks for your attempt anyway.

Reply:
> Never mind :)
> I am getting very close to have this bug fixed. Thanks for your attempt anyway.

// @riophae\vue-treeselect\dist\vue-treeselect.js
// Line 2267
    if (document.activeElement === instance.$refs.menu || instance.$refs.wrapper.contains(document.activeElement)) {
        return this.focus();
    }

Reply:
It does work!